### ISSUE TITLE
Namespace debug styles

### ISSUE THREAD

**Issue by joprice** [OvermindDL1/bucklescript-tea#132]
I'm having issues with the debug overlay's css clashing with some rules from another lib. Would it be a good idea to inline or namespaced them to avoid this?

**Comment by OvermindDL1** [comment]
Makes sense to do, although the state tracking would need to be done in the code side then, not hard to do and little of it.  A PR would be great.  :-)
